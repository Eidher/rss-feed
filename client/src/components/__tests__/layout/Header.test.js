import React from "react";
import renderer from "react-test-renderer";

import Header from "../../layout/Header";

describe("<Header />", () => {
  const wrapper = renderer.create(<Header />);

  it("renders correctly", () => {
    expect(wrapper).toMatchSnapshot();
  });
});
