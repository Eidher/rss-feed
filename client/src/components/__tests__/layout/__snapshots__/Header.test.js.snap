// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Header /> renders correctly 1`] = `
<nav
  className="navbar navbar-expand-lg navbar-light fixed-top"
  id="mainNav"
>
  <div
    className="container"
  >
    <a
      className="navbar-brand js-scroll-trigger"
      href="#page-top"
    >
      <img
        alt="Univision Logo"
        className="logo img-fluid"
      />
       / RSS Feed
    </a>
    <button
      aria-controls="navbarResponsive"
      aria-expanded="false"
      aria-label="Toggle navigation"
      className="navbar-toggler navbar-toggler-right"
      data-target="#navbarResponsive"
      data-toggle="collapse"
      type="button"
    >
      Menu
      <i
        className="fas fa-bars"
      />
    </button>
    <div
      className="collapse navbar-collapse"
      id="navbarResponsive"
    >
      <ul
        className="navbar-nav ml-auto"
      >
        <li
          className="nav-item"
        >
          <a
            className="nav-link js-scroll-trigger"
            href="#download"
          >
            Home
          </a>
        </li>
        <li
          className="nav-item"
        >
          <a
            className="nav-link js-scroll-trigger"
            href="#features"
          >
            RSS Feed
          </a>
        </li>
        <li
          className="nav-item"
        >
          <a
            className="nav-link js-scroll-trigger"
            href="#contact"
          >
            Contact
          </a>
        </li>
      </ul>
    </div>
  </div>
</nav>
`;
